src/reactapp/src/context/Cart/aggregated/actions.js

```diff
@@ -0,0 +1,10 @@
+import { AGGREGATED_CART_DATA } from './types';
+
+export function storeAggregatedCartStatesAction(dispatch, appDispatch, data) {
```

Comment:
We are not using `appDispatch` inside the function. But, we are using the third parameter `data` there. The second parameter is always `appDispatch` here. so that is purposefully added there.

Reply:
I am not sure why this was deemed necessary? What is the purpose of `appDispatch` being the second parameter in cart context actions?

Reply:
In the `sendRequest.js` file used by the API, the section needs a `dispatch` function to show the error messages if a generic error happens. This dispatch is actually appDispatch. So the dispatch of the Cart won't work here. Due to this every cart action will have its first parameter is cartDispatch and the second parameter is appDispatch and then followed by any other params if needed. This way cart actions can pass the appDispatch to the API requests and we will see generic error messages (if the graphql call fails)

Reply:
I am not sure if this setup is optimal in the longterm but its probably out of scope for this MR.. Ok.